pom.xml

```diff
@@ -43,13 +47,11 @@
 
     <properties>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <confluent.common.version>3.2.1-SNAPSHOT</confluent.common.version>
-        <slf4j.version>1.7.21</slf4j.version>
```

Comment:
yeah, starting point. 

we could definitely have things like schema-registry and rest-proxy extend this pom so they can get jetty version (since i doubt anyone else needs that). and move jackson or something else shared up... though we can/will run in to conflicts there.